pom.xml

```diff
@@ -32,32 +32,32 @@
         <ant.version>1.10.12</ant.version>
         <archaius-core.version>0.7.6</archaius-core.version>
         <assertj-guava.version>3.4.0</assertj-guava.version>
-        <byte-buddy.version>1.12.1</byte-buddy.version>
+        <byte-buddy.version>1.12.4</byte-buddy.version>
         <caffine.version>3.0.3</caffine.version>
-        <checker-qual.version>3.10.0</checker-qual.version>
+        <checker-qual.version>3.20.0</checker-qual.version>
         <classmate.version>1.5.1</classmate.version>
         <commons-configuration.version>1.10</commons-configuration.version>
-        <commons-codec.version>1.14</commons-codec.version>
+        <commons-codec.version>1.15</commons-codec.version>
         <commons-io.version>2.11.0</commons-io.version>
         <commons-lang.version>2.6</commons-lang.version>
         <commons-lang3.version>3.12.0</commons-lang3.version>
         <commons-logging.version>1.2</commons-logging.version>
         <consul-client.version>1.5.1</consul-client.version>
         <curator.version>5.1.0</curator.version>
-        <dropwizard.version>2.0.25</dropwizard.version>
-        <dropwizard.metrics.version>4.1.25</dropwizard.metrics.version>
+        <dropwizard.version>2.0.26</dropwizard.version>
```

Comment:
this needs to be DW 2.0.27

Reply:
...and since this is 2.0.26, does that imply you might have checked the versions against DW 2.0.26 instead of 2.0.27?